docs/streams/stream-basics/subscriptions.md

```diff
@@ -1,84 +1,73 @@
 ---
-sidebar_position: 40
+sidebar_position: 200
 title: Subscriptions
 ---
 
-A subscription is a named configuration rule that determines how messages are delivered to consumers. There are three available subscription modes in GDN streams: [exclusive](#exclusive), [shared](#shared), and [failover](#failover). These modes are illustrated in the figure below.
+A subscription is a named configuration rule that determines how messages are delivered to consumers. There are three available subscription modes in GDN streams: [exclusive](#exclusive), [shared](#shared), [failover](#failover), and [key_shared](#key_shared). These modes are illustrated in the figure below.
 
 ![stream-subscription-modes](/img/stream-subscription-modes.png)
 
+To configure a subscription:
+
+1. Create a [producer](producers.md).
+1. Create at least two [consumers](consumers.md) with the same subscription name. For example, `consumer-subscription`.
+
+To test the example code, open three terminals simultaneously and run `node producer.js`, then run `consumer-1.js` in second terminal and `consumer-2.js` in third terminal. If successful, you will see messages in both consumer terminals.
+
 ## Exclusive
 
-In _exclusive_ mode, only a single consumer is allowed to attach to the subscription. If more than one consumer attempts to subscribe to a stream using the same subscription, then the consumer receives an error.
+Exclusive mode is the default in which only one consumer is allowed to subscribe. If further consumers attempt to subscribe to a stream with the same subscription, then the consumer receives an error.
 
 In the diagram below, only Consumer-A is allowed to consume messages.
 
-Exclusive mode is the default subscription mode.
-
 ![stream-exclusive-subscriptions](/img/stream-exclusive-subscriptions.png)
 
 ## Shared
 
-In _shared_ or _round robin_ mode, multiple consumers can attach to the same subscription. Messages are delivered in a round robin distribution across consumers, and any given message is delivered to only one consumer.
+In shared mode (also referred to as _round robin_), messages are distributed across consumers so that any given message is delivered to only one consumer.
```

Comment:
```suggestion
In shared mode, also referred to as _round robin_, messages are distributed across consumers so that any given message is delivered to only one consumer.
```